package.json

```diff
@@ -15,7 +15,10 @@
     "start": "react-scripts start",
```

Comment:
Please add project title and project descriptions in README. You may use [this](https://github.com/microverseinc/readme-template) template from Microverse and remember to customize it according to your project.